synced_versions_formulae.json

```diff
@@ -28,7 +28,6 @@
   ["git", "git-credential-libsecret", "git-gui", "git-svn"],
   ["hdf5", "hdf5-mpi"],
   ["ilmbase", "openexr@2"],
-  ["libexosip", "libosip"],
```

Comment:
there is no `libexosip 5.3.1` yet, removing this binding.

Reply:
Maybe we should ask upstream if they're supposed to be synced

Reply:
> The reason for osip 5.3.1 is a CVE
And this time, I make an exception: there will be no 5.3.1 release "at the same time".

from Aymeric (the maintainer)

---

I think we can move forward with this change